distribution/openrct2.d.ts

```diff
@@ -2115,6 +2115,12 @@ declare global {
          * The min chain lift speed for this ride in miles per hour.
          */
         readonly minLiftHillSpeed: number;
+
+        /**
+         * The satisfaction rating of the ride.
+         * Times it by 5 to get the percentage out of 100.
+         */
+        satisfaction: number;
```

Comment:
I believe this is commonly exposed as a decimal number from 0..1, but I’d like @Sadret to confirm.

In general, you don’t want to expose too much of the internal representation to the plugins.

Reply:
I would compare this to `Ride.downtime` which is 0 to 100, i.e. percentage. Unfortunately we are not super consistent here, but 0 to 20 is definitely not the way to go.

Reply:
How about other things that are percentages? Are those exposed as `0..1`?

Reply:
Where else are percentages exposed? Floats in general are rather rare in the API.

Reply:
Right, you requested using 0..1 in [my PR](https://github.com/OpenRCT2/OpenRCT2/pull/22069#discussion_r1606933856), so I was initially under the impression that this was common already.

I guess that if we exposed ride reliability as 0..100, then it would make sense to use that throughout.